src/sidebar/services/annotations.ts

```diff
@@ -46,10 +61,12 @@ export class AnnotationsService {
   private _applyDraftChanges(annotation: Annotation): Annotation {
     const changes: Partial<Annotation> = {};
     const draft = this._store.getDraft(annotation);
+    const authority =
+      this._settings.services?.[0]?.authority ?? this._store.defaultAuthority();
```

Comment:
I'm not 100% sure this is the right way to get the authority to be used for mentions. Perhaps getting it from current user would be more correct.

Reply:
At the point when this is invoked there should always be a logged-in user ID, so I would get it from there. We have functions to parse the authority from the user ID I believe.